pkg/cmd/pulumi/import.go

```diff
@@ -542,6 +563,14 @@ func newImportCmd() *cobra.Command {
 				if err != nil {
 					return result.FromError(err)
 				}
+
+				// Write the file we've built out to the local directory so if there's any issues users can manually
+				// edit the file and try again with --file
+				path, err := writeImportFile(f)
+				if err != nil {
+					return result.FromError(err)
+				}
+				pCtx.Diag.Infof(diag.Message("", "Import file written to %s"), path)
```

Comment:
I feel this should be more detailed, like telling the user about --file and why they might want this, but not sure on a clean and succinct phrasing for it.

Reply:
Can we say something along those lines only if the import fails?